README.md

```diff
@@ -1 +1,165 @@
+
+Video Presntation link(https://drive.google.com/file/d/1tBGtpw2K507swHiRJKkgwyR2mdt-b4Kj/view?usp=sharing)></a>
+
+
+**[Catalog of my things]** is based on Ruby Language.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href=#>RUBY</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href=#>N/A</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href=#>Local Storage</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Create classes and inheritence added]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+- clone this https://github.com/QueenterJuma/ruby_catalog.git
+
+
+### Prerequisites
+
+In order to run this project you need:
+<ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+  
+### Setup
+
+Clone this repository to your desired folder:
+<ul>
+<li>cd my-folder</li>
+<li>git clone https://github.com/QueenterJuma/ruby_catalog.git</li>
+</ul>
+
+### Install
+
+Install this project with:
+<ul>
+<li>bundle install</li>
+</ul>
+
+### Usage
+
+To run the project, execute the following command:
+<ul>
+<li>ruby main.rb</li>
+</ul>
+
+### Run tests
+
+To run tests, run the following command: Install through : gem install rspec
+
+Run: rspec to tun tests
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="DHEERAJ SACHDEVA"></a>
+
+👤 **Queenter Juma**
+
+- GitHub: [@QueenterJuma](https://github.com/QueenterJuma)
+
+👤 **Heldricks Adhola**
+
+- GitHub: [@Adholah96](https://github.com/Adholah96)
+
+👤 **DHEERAJ SACHDEVA**
+
+- GitHub: [@githubhandle](https://github.com/dheerajsachdeva)
+- Twitter: [@twitterhandle](https://twitter.com/dheerajarya)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/dheeraj-sachdeva-502b2b8/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[More conditional checking and features will be added in future]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [repo:https://github.com/QueenterJuma/ruby_catalog.git]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please follow me on github
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
```

Comment:
- [OPTIONAL] Since you added the content of this section, let's kindly remove the word OPTIONAL

Reply:
Done. Thanks